### DISCUSSION TITLE
Thoughts on Integrating with Refine

### DISCUSSION THREAD

**Discussion by NicoPowers** [cloudscape-design/components#1466] [General]
I love the cloudscape design system, you all have done such a great job on the quality of components, great documentation, customization/styling, and I can see this UI exploding as more people find out about it.

I have been using Refine for a little bit too for some other projects, and love it's ability to generate CRUD pages for resource that make it so much faster and easier to develop dashboard, admin UIs, and more.

They currently support MUI and the like, but I thought it would be game changer if there was cloudscape design integration; it would look and feel far superior than the rest, especially with the dashboard component. 

An integration with refine would also substantially increase the user base of cloudscape-design; what are some of your thoughts on this?

**Comment by just-boris** [comment]
Hello!

We have no use-case for such workflow in our team directly, so this integration is not our priority.

Also, it seems like this integration can be done on the user land. Feel free to explore
